sites/fast-website/src/public/index.html

```diff
@@ -3,12 +3,114 @@
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
   <body>
     <fast-design-system-provider use-defaults>
-      <fast-button>Hello, World!</fast-button>
-      <gradient-picker>stub</gradient-picker>
+      <div class="wrapper">
+        <!--header-->
+        <header class="header">
+          <div>
+              <button>Logo</button>
+          </div>
+          <div class="header-link-group">
+            <button>Component</button>
+            <button>Integration</button>
+            <button>Documentation</button>
+            <button>Community</button>
+          </div>
+          <div>
+              <button>GitHub</button>
+          </div>
+        </header>
+        <!--end of header-->
+
+        <!-- hero section-->
+        <section class="page-section-container">
```

Comment:
I think the "page" aspect is un-needed, also "container".

Reply:
updated class name